pkg/operation/seed/seed.go

```diff
@@ -75,12 +74,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
-type byName []corev1.ConfigMap
```

Comment:
Thanks for cleaning this up! Actually, I wanted to do this as well while introducing the generic sorter, but then forgot about it.